### PR TITLE
Add wojtekt to k8s-infra viewers

### PR DESCRIPTION
**Pull request by wojtek-t**
/assign @aojea @ameukam

### CODE REVIEW

File: groups/sig-k8s-infra/groups.yaml [hunk 1]
```diff
@@ -203,6 +203,7 @@ groups:
       - cblecker@gmail.com
       - davanum@gmail.com
       - nikhitaraghunath@gmail.com
+      - wojtekt@google.com
```
Comment (ameukam) [inline groups/sig-k8s-infra/groups.yaml:206]: @wojtek-t why this is needed ? How the redirect is related to integration job failures ?

Comment (aojea) [reply groups/sig-k8s-infra/groups.yaml:206]: I know that I'm able to see the logs of the failures but don't know which entry is the one that allows me to do that, so I suggested to add himself to the same groups I belong XD

Comment (aojea) [reply groups/sig-k8s-infra/groups.yaml:206]: image

Comment (ameukam) [reply groups/sig-k8s-infra/groups.yaml:206]: Only membership of `k8s-infra-prow-viewers@kubernetes.io` is required. And @wojtek-t is supposed to be able to see metrics of this project. 
- https://github.com/kubernetes/k8s.io/blob/main/groups/sig-scalability/groups.yaml#L53-L72
- https://github.com/kubernetes/k8s.io/blob/b76a417a89b46ecc760c7a73472c2d46b39aa2f2/groups/sig-testing/groups.yaml#L144
https://console.cloud.google.com/monitoring/dashboards?project=k8s-infra-prow-build&pageState=(%22dashboards%22:(%22t%22:%22All%22))

Comment (wojtek-t) [reply groups/sig-k8s-infra/groups.yaml:206]: This is super strange...

I've seen that, but I was trying to access that project an hour ago and didn't have access.
But I tried now and I have access - I have no idea what happened.

Anyway - closing this PR for now.

**Pull request closed without merging**